src/Symfony/Component/Security/Http/Firewall/SwitchUserListener.php

```diff
@@ -142,9 +143,15 @@ private function attemptSwitchUser(Request $request, string $username): ?TokenIn
         if (null !== $originalToken) {
             if ($token->getUsername() === $username) {
                 return $token;
+            } elseif (!$this->allowAlreadySwitched) {
+                $e = new AlreadySwitchedException(sprintf('You are already switched to "%s" user.', $token->getUsername()));
```

Comment:
What about adding this exception in 3.4 as a bug fix?

Reply:
I think this should be considered a feature and not a bug fix. (in the issue, we even argued whether or not this is a BC break)

Reply:
I still don't see any BC about changing the exception here.